src/Symfony/Component/RateLimiter/RateLimiterFactory.php

```diff
@@ -82,7 +82,7 @@ protected static function configureOptions(OptionsResolver $options): void
             ->define('limit')->allowedTypes('int')
             ->define('interval')->allowedTypes('string')->normalize($intervalNormalizer)
             ->define('rate')
-                ->default(function (OptionsResolver $rate) use ($intervalNormalizer) {
+                ->nested(function (OptionsResolver $rate) use ($intervalNormalizer) {
```

Comment:
`options(...)` ? (together with `setOptions()`)
or
`nestOptions(...)` ? (together with `setNestedOptions()`)

Reply:
I like the first alternative, it's simple (just one word), but `isNested()` already exist, so could it be confusing? `options()` + `setOptions()` + `isNested()`?

Reply:
I did the update in the meantime

Reply:
/cc @symfony/mergers any other name suggestions for this new method?